consumers/google.go

```diff
@@ -65,10 +56,21 @@ func NewGoogleDNS() (Consumer, error) {
 		return nil, fmt.Errorf("Error creating DNS service: %v", err)
 	}
 
+	resp, err := client.ManagedZones.List(params.project).Do()
```

Comment:
Yes, it should, but this requires a little more thought.

Would you run it per `sync` and then share a structure with `Process`? Or would you run a separate Goroutine?